### PR TITLE
Add fluxcd-source-controller v1.1.2

### PR DESCRIPTION
**Pull request by hansrajdas**
Add fluxcd-source-controller v1.1.2

### CODE REVIEW

File: packages/fluxcd-source-controller/1.1.2/package.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  name: fluxcd-source-controller.community.tanzu.vmware.com.1.1.2
+spec:
+  refName: fluxcd-source-controller.community.tanzu.vmware.com
+  version: 1.1.2
+  releaseNotes: "Releasing TAP compatible source controller"
```
Comment (chaudharyalka) [inline packages/fluxcd-source-controller/1.1.2/package.yaml:8]: We should update the release not here. This is not compatible with tap.

File: packages/fluxcd-source-controller/1.1.2/package.yaml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+          description: "Kubernetes version of the cluster"
+        ca_bundle:
+          type: string
+          default: ""
+          description: "a PEM-encoded CA certificate. This will be used to connect to the registry."
+  template:
+    spec:
+      fetch:
+      - imgpkgBundle:
+          image: projects.registry.vmware.com/tce/fluxcd-source-controller-bundle@sha256:b095202828e66e3561ddc5c642bcdb10ab3d944c159e662abb567698edb0280d
```
Comment (chaudharyalka) [inline packages/fluxcd-source-controller/1.1.2/package.yaml:66]: I hope this the updated image referring to version 1.1.2

**Review by chaudharyalka — COMMENTED** [review]
I would suggest better to add this package to some dummy repo  then install this package and check everything works as expected before merging.

File: packages/fluxcd-source-controller/1.1.2/package.yaml (hunk 2 above)
Comment (hansrajdas) [reply packages/fluxcd-source-controller/1.1.2/package.yaml:66]: Yes, this is updated

File: packages/fluxcd-source-controller/1.1.2/package.yaml (hunk 1 above)
Comment (hansrajdas) [reply packages/fluxcd-source-controller/1.1.2/package.yaml]: Updated

Comment (hansrajdas) [reply packages/fluxcd-source-controller/1.1.2/package.yaml]: > I would suggest better to add this package to some dummy repo then install this package and check everything works as expected before merging.

That's done, by referring feature branch from cloned repo and creating packages & repo using VMware internal pipelines

**Pull request merged**